docs/src/add-services/varnish.md

```diff
@@ -146,8 +144,12 @@ This limitation may be lifted in the future.
 
 ## Stats endpoint
 
-The Varnish service also offers an `http+stats` endpoint, which provides access to some Varnish analysis and debugging tools.
-To access it, from a dedicated app container add the following to your [app configuration](../create-apps/app-reference.md):
+The Varnish service also offers an `http+stats` endpoint,
+which provides access to some Varnish analysis and debugging tools.
+
+You can't use it from an app fronted by Varnish because of the restriction with [circular relationships](#circular-relationships).
+To access the stats, create a **separate app** with a relationship *to* Varnish, but not *from* it.
+Add the following to your [app configuration](../create-apps/app-reference.md):
 
 {{< readFile file="src/registry/images/examples/full/varnish.app.yaml" highlight="yaml" location=".platform.app.yaml" >}}
```

Comment:
:hammer_and_wrench: Why not provide a working example to make the page easier to use?
Something along the lines of:
```
name: varnishstats

type: "php:8.1"

build:
    flavor: none

relationships:
    varnishstats: "varnish:http+stats"
```

Reply:
I was focusing on removing the bug, rather than optimizing the flow of building the workaround. I was also hoping to get an answer on when the product would be fixed so the workaround wasn't necessary, so I didn't spend so long on the flow there. I wasn't sure this was a comprehensive or clear list of endpoints.

I can add an example, though.